desktop/updater/lib/Updater.js

```diff
@@ -94,7 +94,7 @@ class Updater extends EventEmitter {
     let text = originalText;
 
     for (const key in macros) {
-      if (macros.hasOwnProperty(key)) {
+      if (Object.prototype.hasOwnProperty.call(macros, key)) {
```

Comment:
what did the linter not like about this?

did we even write this code or have we pulled it in from a dependency?

Reply:
https://eslint.org/docs/rules/no-prototype-builtins

Reply:
I can't recall ever having had bugs from this violation and it looks like it makes the code significantly more noisy to provide that safety. Also, in this case that it fixes we're absolutely _not_ violating any safety concerns as is possible to verify by inspection.

Do you have strong feelings about this rule? I'd be inclined to turn it off.